reactor-core/src/main/java/reactor/core/publisher/Flux.java

```diff
@@ -7076,6 +7080,31 @@ public int getPrefetch() {
 				Queues.get(prefetch));
 	}
 
+	public final Flux<T> prefetch() {
+		return prefetch(Queues.SMALL_BUFFER_SIZE);
+	}
+
+	public final Flux<T> prefetch(boolean lazyMode) {
+		return prefetch(Queues.SMALL_BUFFER_SIZE, lazyMode);
+	}
+
+	public final Flux<T> prefetch(int prefetch) {
+		return prefetch(prefetch, false);
+	}
+
+	public final Flux<T> prefetch(int prefetch, int lowTide) {
+		return prefetch(prefetch, lowTide, false);
+	}
+
+	public final Flux<T> prefetch(int prefetch, boolean lazyMode) {
+		return prefetch(prefetch, prefetch, lazyMode);
+	}
+
+	public final Flux<T> prefetch(int prefetch, int lowTide, boolean lazyMode) {
```

Comment:
nit: for consistency with the `lowTide` parameter, I would rename `prefetch` to `highTide` here